docs/cve-creation-guide.md

```diff
@@ -0,0 +1,189 @@
+# CVE Creation Guide
+This is a guide to creating a CVE request via Mitre.
+It may be of some use to explain what the CVE fields are looking for with some hints and examples.
+
+# Creating a CVE Request
+
+This is an example of how to create a CVE with an older vulnerability from Mojolicious that did not have one previously assigned.
+
+  * CPANSA-Mojolicious-2018-01
+    Mojo::UserAgent::CookieJar leaks old cookies because of the missing host_only flag on empty domain.
+
+    Affected range: <7.66
+
+    CVEs: 
+
+    References:
+    https://github.com/mojolicious/mojo/pull/1192
+    https://github.com/mojolicious/mojo/issues/1185
+    https://github.com/mojolicious/mojo/commit/c16a56a9d6575ddc53d15e76d58f0ebcb0eeb149
+
+## Research the CVE database to ensure it has not been logged previously
+
+Ensuring that the issue has not been previously logged is very important.
+Duplicate entries will merely serve to confuse the issue and polute the CVE database.
+
+1. Access [Mitre.org CVE Search](https://cve.mitre.org/cve/search_cve_list.html)
+1. Search for various keywords including but not limited to the distribution or module name and any applicable words.
+Review the results
+1. If the CVE appears to be the correct one it would likely be best to submit an issue containing the CVE number and link to the CVE database to  [https://github.com/briandfoy/cpan-security-advisory/issues](https://github.com/briandfoy/cpan-security-advisory/issues)
+
+## Access Mitre CVE Form
+
+1. Access [Mitre CVE Form](https://cveform.mitre.org/)
+1. Select a request type: **_Report Vulnerability/Request CVE ID_**
+1. Enter your e-mail address: **_cpan-security@perl.org_**
```

Comment:
Authors/reporters should prob use their own email address here and not our list

Reply:
True